src/parse.c

```diff
@@ -4440,7 +4440,7 @@ void Parser::checkCstyleTypeSyntax(Loc loc, Type *t, int alt, Identifier *ident)
 
 Statement *Parser::parseStatement(int flags, const utf8_t** endPtr, Loc *pEndloc)
 {
-    Statement *s;
+    Statement *s = NULL;
```

Comment:
How is it a false positive? If I follow the code correctly, at line 5057, when token is `TOKversion`, and next token is `TOKAssign`, `s` is not initialized.

Is this a bug?

Can we do `(Statement *)(void *)1`, would that translate to valid D?